content/en/blog/releases/Kitex/release-v0_12_0.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Kitex Release v0.12.0"
+linkTitle: "Release v0.12.0"
+projects: ["Kitex"]
+date: 2025-01-13
+description: >
+---
+
+## **Introduction to Key Changes**
+
+### Simplified Product Recommendation - Remove Apache Thrift Dependency
+We strongly recommend removing Apache Codec to resolve the compilation issues caused by Apache's incompatible changes and to **reduce the product size by 50%**. 
+Please replace it with Kitex's own Thrift codec: FastCodec, which does not rely on Apache Thrift Codec.
+Future version plans: Kitex will remove Apache products by default. User guide: [Kitex Remove Apache Thrift User Guide](/docs/kitex/best-practice/remove_apache_codec)
+
+### New Features
+1. **Thrift Streaming over TTHeader - Custom Streaming Protocol**: Supported streaming calls based on the TTHeader protocol, optimizing stability and performance issues caused by the high complexity of the gRPC streaming protocol. For more details: [Streamx](/docs/kitex/tutorials/basic-feature/streamx/) 
```

Comment:
Although performance optimization is the goal, there is currently no performance advantage, _optimizing stability issues_ is better